### PR TITLE
Update rdm-visualisation.md

### LINKED ISSUE #3441: More Resources For Data Visualization Chapter
Additional Resources and references for the Data Visualization chapter
https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-visualisation.html#data-visualisation

### Training & Workshops

-   

### Books

-   [Building Science Graphics](https://openlibrary.org/works/OL28353366W/Building_Science_Graphics) - I leave my copy of this one in the fishbowl if anyone wants to borrow it
-   [A Field Guide to Digital Color](https://openlibrary.org/works/OL6130168W/A_Field_Guide_to_Digital_Color)
-   [The Grammar of Graphics](https://openlibrary.org/works/OL18235300W/The_grammar_of_graphics)
-   [The Visual Display of Quantitative Information](https://openlibrary.org/works/OL2824012W/The_Visual_Display_of_Quantitative_Information)

### Podcasts

-   [Data Viz Today](https://dataviztoday.com/)
-   [Data Stories](https://datastori.es/)
-   [PolicyViz](https://policyviz.com/podcast/)

### Video

-   Outline of grammar of graphics
    -   [A Grammar of Graphics](https://www.youtube.com/watch?v=RCaFBJWXfZc) - Excellent summary of the grammar of graphics layers or 'functional pipeline'
    -   [Leland Wilkinson - The Grammar of Graphics](https://www.youtube.com/watch?v=1X93Sum_SyM) - Leland himself giving a quick high level summary of the grammar of graphics
-   [Martin Krzywinski: A pandemic of bad charts](https://www.youtube.com/watch?v=_YGmfsKL8N8)
-   [EMBL Keynote Lecture 2019 - Data visualization and data science, Hadley Wickham](https://www.youtube.com/watch?v=9YTNYT1maa4)
-   Practical high level intros to Tufte's principles
    -   [What should we do to improve our graphics and figures?](https://www.youtube.com/watch?v=00Fha1lkRxk)
    -   [Graphical Excellence](https://www.youtube.com/watch?v=VkyzSAPkQ50)

### Organizations

-   [biovis](http://biovis.net/)

### Websites & Blogs

-   [A Comprehensive Guide to the Grammar of Graphics for Effective Visualization of Multi-dimensional Data](https://towardsdatascience.com/a-comprehensive-guide-to-the-grammar-of-graphics-for-effective-visualization-of-multi-dimensional-1f92b4ed4149) - a shorter more accessible introduction to the grammar of graphics than Leland's book or Hadley's paper

### Papers and other scholarly publications

-   [A layered grammar of graphics](http://dx.doi.org/10.1198/jcgs.2009.07098) [@wickham2010]

Hi @EstherPlomp I thought that these might make some good additions what you've been working on

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Fixes #3441

### List of changes proposed in this PR (pull-request)

* add data visualisation resources
* add image 

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by srtee** [comment]
We could also improve some existing lines -- the "upset graphs" and "Python plotly" dot points don't give much context. Maybe we could expand those to:

> * Upset graphs are a straightforward way to visualize set intersections in a matrix layout, which can help in analysing multiple datasets at once.
> * In Python Plotly, annotations and animations can  serve as extra visual cues to highlight important features.

File: book/website/reproducible-research/rdm/rdm-visualisation.md [hunk 1]
```diff
@@ -39,16 +49,36 @@ Below follow some resources that may help you in creating clearer, more accessib
...
 - [Creating clear and informative image-based figures for scientific publications](https://doi.org/10.1371/journal.pbio.3001161) by {cite:ps}`Jambor2021dataviz`
+- [A layered grammar of graphics](http://dx.doi.org/10.1198/jcgs.2009.07098) by {cite:ps}`Wickham2010dataviz`
 
 ### Other text resources and examples
 
 - [Data Visualisation Course](https://exploratory-data-visualization.netlify.app/)
 - [Free visual and communication resources](https://padlet.com/evansemporiumstore/lks3aoyyhkpnkmpe)
 - [A collection of notebooks demonstrating plotting with matplotlib](https://github.com/KirstieJane/NH19-Visualization/)
+- [A Comprehensive Guide to the Grammar of Graphics for Effective Visualization of Multi-dimensional Data](https://towardsdatascience.com/a-comprehensive-guide-to-the-grammar-of-graphics-for-effective-visualization-of-multi-dimensional-1f92b4ed4149)
+- [Friends Don't let Friends...](https://github.com/cxli233/FriendsDontLetFriends)
```
Comment (kallewesterling) [inline book/website/reproducible-research/rdm/rdm-visualisation.md:60]: The full title is "Friends Don't Let Friends Make Bad Graphs" and they have DOI: [10.5281/zenodo.7097522](https://dx.doi.org/10.5281/zenodo.7097522).

**Review by kallewesterling — CHANGES_REQUESTED** [review]
This looks great to me -- loving the new graphic from Scriberia and everything. I've added a comment about the resource I had suggested, but that's all that I suggest changing.

**Comment by RichardJActon** [comment]
Whilst not open access per se these three books are all readily available:

- [A Field Guide to Digital Color](https://openlibrary.org/works/OL6130168W/A_Field_Guide_to_Digital_Color)
- [The Grammar of Graphics](https://openlibrary.org/works/OL18235300W/The_grammar_of_graphics)
- [The Visual Display of Quantitative Information](https://openlibrary.org/works/OL2824012W/The_Visual_Display_of_Quantitative_Information)

(A DRM free copy of each has found its way into my digital library at any rate.)

I'd consider Grammar of graphics a extremely important recommended reading to anyone implementing a graphics library or user interface for creating visualizations. Leland Wilkinson is, arguably, to the language of expressing graphics as Noam Chomsky is to linguistics. If you implement a graphics library or UI without reading at least some of grammar of graphics first you (and your users) **WILL** regret it. The higher level summaries that are linked to are enough for most users. Anyone who spends any significant amount of time thinking about composing data visualisations, could benefit from reading at least some of it. It's pretty dense I've not read it cover-to-cover but its been extremely useful for developing a framework for thinking and reasoning about data visualization.

The Visual Display of Quantitative Information is worth referencing just because of how incredibly influential Tufte's views have been on the field - it's the origin for so many of the recommendations you encounter in day to day practice.

File: book/website/reproducible-research/rdm/rdm-visualisation.md (hunk 1 above)
Comment (EstherPlomp) [reply book/website/reproducible-research/rdm/rdm-visualisation.md:60]: Oh very nice - I overlooked that. Guess I now have to add it to the bibliography hehe

**Comment by EstherPlomp** [comment]
> We could also improve some existing lines -- the "upset graphs" and "Python plotly" dot points don't give much context. Maybe we could expand those to:
> 
> > * Upset graphs are a straightforward way to visualize set intersections in a matrix layout, which can help in analysing multiple datasets at once.
> > * In Python Plotly, annotations and animations can  serve as extra visual cues to highlight important features.

Thank you @srtee! That is a great add!

**Comment by EstherPlomp** [comment]
> Whilst not open access per se these three books are all readily available:
> 
>     * [A Field Guide to Digital Color](https://openlibrary.org/works/OL6130168W/A_Field_Guide_to_Digital_Color)
> 
>     * [The Grammar of Graphics](https://openlibrary.org/works/OL18235300W/The_grammar_of_graphics)
> 
>     * [The Visual Display of Quantitative Information](https://openlibrary.org/works/OL2824012W/The_Visual_Display_of_Quantitative_Information)
> 
> 
> (A DRM free copy of each has found its way into my digital library at any rate.)
> 
> I'd consider Grammar of graphics a extremely important recommended reading to anyone implementing a graphics library or user interface for creating visualizations. Leland Wilkinson is, arguably, to the language of expressing graphics as Noam Chomsky is to linguistics. If you implement a graphics library or UI without reading at least some of grammar of graphics first you (and your users) **WILL** regret it. The higher level summaries that are linked to are enough for most users. Anyone who spends any significant amount of time thinking about composing data visualisations, could benefit from reading at least some of it. It's pretty dense I've not read it cover-to-cover but its been extremely useful for developing a framework for thinking and reasoning about data visualization.
> 
> The Visual Display of Quantitative Information is worth referencing just because of how incredibly influential Tufte's views have been on the field - it's the origin for so many of the recommendations you encounter in day to day practice.

Thanks @RichardJActon - I added the three books in your reply now 👍

File: book/website/reproducible-research/rdm/rdm-visualisation.md [hunk 2]
```diff
@@ -11,26 +11,36 @@ Therefore, data visualisation is an aspect of research data management!
...
+-->
+
 (rr-rdm-visualisation-tools)=
 ## Tools
 
-- [upset graphs](https://upset.app/)
-- Python plotly
-  - [annotating visualisations in python plotly](https://medium.com/nerd-for-tech/enriching-data-visualizations-with-annotations-in-plotly-using-python-6127ff6e0f80) (blog and video)
+-  [upset graphs](https://upset.app/) are a straightforward way to visualize set intersections in a matrix layout, which can help in analysing multiple datasets at once.
+-  In Python Plotly, annotations and animations can serve as extra visual cues to highlight important features.
```
Comment (BeastyBlacksmith) [inline book/website/reproducible-research/rdm/rdm-visualisation.md:28]: Is that really specific to python plotly? Should definetely work also in javascript. Not sure what the state of the other backends is

Comment (EstherPlomp) [reply book/website/reproducible-research/rdm/rdm-visualisation.md]: Fair point - I rewrote it to reflect that this is pointing things out for Plotly and not speaking in general!

**Comment by EstherPlomp** [comment]
An approving review of this PR would be nice - please let me know if there's anything that still needs to be changed!

**Review by BeastyBlacksmith — APPROVED** [review]

File: book/website/reproducible-research/rdm/rdm-visualisation.md [hunk 3]
```diff
@@ -11,26 +11,36 @@ Therefore, data visualisation is an aspect of research data management!
...
+---
+name: project-design
+alt: A group of people collaboratively developing a data visualisation by using a giant pencil. 
+---
+ _The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+-->
+
 (rr-rdm-visualisation-tools)=
 ## Tools
```
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply book/website/reproducible-research/rdm/rdm-visualisation.md:25]: Added! Let me know if you know more :)

**Review by kallewesterling — APPROVED** [review]
I think this is a great addition to the Turing Way!

**Comment by EstherPlomp** [comment]
Thank you all! Will merge now - I was waiting for some extra resources but we can open up another issue/PR for that once those come through :)

**Pull request merged**